src/protect-function-app/Api.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <AzureFunctionsVersion>v3</AzureFunctionsVersion>

```

Comment:
Can you update to v4?  This is the [Microsoft recommended version specifically for .NET 6 applications](https://docs.microsoft.com/azure/azure-functions/functions-versions?tabs=in-process%2Cv4&pivots=programming-language-csharp) and is generally the new default recommended version for most other workloads.

Reply:
thanks for feedback @ckittel 

done | addressed from [8939b](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/19/commits/8939b9905910a861f1cd32883ac6f96a8f79cdce)